zerver/lib/actions.py

```diff
@@ -4433,6 +4436,9 @@ def do_revoke_user_invite(prereg_user: PreregistrationUser) -> None:
                                 object_id=prereg_user.id).delete()
     prereg_user.delete()
     clear_scheduled_invitation_emails(email)
+    event = dict(type="invites_changed")
+    admin_ids = [user.id for user in prereg_user.realm.get_admin_users()]
+    send_event(event, admin_ids)
```

Comment:
We should add some logic in `zerver/lib/events.py` and a test in `test_events.py` for this event.  I think it'll be like typing notifications where it's basically ignored, but it's important to have full test coverage of our `send_event` calls in those files.

Reply:
Oh, I guess those are in the second commit; we should squash.